MediaBrowser.Model/Syncplay/SendCommandType.cs

```diff
@@ -0,0 +1,21 @@
+namespace MediaBrowser.Model.Syncplay
+{
+    /// <summary>
+    /// Enum SendCommandType.
+    /// </summary>
+    public enum SendCommandType
+    {
+        /// <summary>
+        /// The play command. Instructs users to start playback.
+        /// </summary>
+        Play = 0,
+        /// <summary>
+        /// The pause command. Instructs users to pause playback.
+        /// </summary>
+        Pause = 1,
+        /// <summary>
+        /// The seek command. Instructs users to seek to a specified time.
+        /// </summary>
+        Seek = 2
```

Comment:
how do I request a "next item" or just "different item than is currently played"?

Reply:
There's no way at the moment. Changing the current item means creating a new group. In the future this will be handled by the shared playlist I guess.